examples/adpt.hotspot.11.jdkw

```diff
@@ -0,0 +1,6 @@
+JDKW_RELEASE=latest
+JDKW_DIST=adpt
+JDKW_BUILD=10
+JDKW_VERSION=11.0.5
+JVM_IMPL=hotspot
```

Comment:
This new flag will need to be documented.

Reply:
Added docs